build-scripts/for-changelog/svn-changelog.awk

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This script is necessary for generating the changelog from SVN and is not required for build at github. I'd removed it from the PR